basilisk/tests/test_main_app.py

```
@@ -0,0 +1,280 @@
+        assert main_app.validate_config(invalid_config) is False
+
+# Test utilities and cleanup
+def teardown_module():
+    """Clean up after all tests in this module."""
+    import tempfile
+    import shutil
+    temp_dirs = [d for d in os.listdir(tempfile.gettempdir()) if d.startswith('pytest')]
+    for temp_dir in temp_dirs:
+        try:
+            shutil.rmtree(os.path.join(tempfile.gettempdir(), temp_dir))
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)